bosh_cli/lib/cli/commands/ssh.rb

```diff
@@ -6,7 +6,7 @@ module Command
     class Ssh < Base
 
       # bosh ssh
-      usage 'ssh'
+      usage 'ssh <job>/<index>'
```

Comment:
I'm not 100% sure, but with the changes in place, things like `bosh help ssh` seems to work...

Is there a way I can test that?